dataprofiler/tests/profilers/test_numerical_options.py

```diff
@@ -0,0 +1,105 @@
+from dataprofiler.profilers.profiler_options import BooleanOption, NumericalOptions
+from dataprofiler.tests.profilers.test_base_column_options import TestBaseColumnOptions
+
+class TestNumericalOptions(TestBaseColumnOptions):
+    
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.keys += ["min", "max", "sum", "variance", "histogram_and_quantiles"]
+        cls.option_class = NumericalOptions
```

Comment:
we will have to validate these pass when conducted in parallel.
try running: `python3 -m pytest dataprofiler/tests/ --cov=dataprofiler --cov-fail-under=80 --cov-report=xml:dist/coverage.xml --forked`

Reply:
The keys are causing issues when running in parallel - I will try to resolve and update the PR